code/modules/mob/living/silicon/ai/freelook/eye.dm

```diff
@@ -104,3 +106,11 @@
 		return //won't work if dead
 	acceleration = !acceleration
 	usr << "Camera acceleration has been toggled [acceleration ? "on" : "off"]."
+
+
+/mob/camera/aiEye/Hear(message, atom/movable/speaker, message_langs, raw_message, radio_freq, list/spans)
+	if(relay_speech && speaker && ai && !radio_freq && speaker != ai && near_camera(speaker))
+		raw_message = ai.lang_treat(speaker, message_langs, raw_message, spans)
+		var/name_used = speaker.GetVoice()
+		var/rendered = "<i><span class='game say'>Enhanced Camera: <span class='name'>[name_used]</span> <span class='message'>[raw_message]</span></span></i>"
+		ai.show_message(rendered, 2)

```

Comment:
I tried doing it a few days ago and it didn't work, I probably passed the arguements wrong. I'll try again